src/textual/__init__.py

```diff
@@ -3,6 +3,7 @@
 from rich.console import RenderableType
 
 __all__ = ["log", "panic"]
+__version__ = "0.1.15"
```

Comment:
I don't find this particularly clean but it is a commonly used pattern in a bunch of libraries from what I've read. It corresponds to a now rejected PEP: https://peps.python.org/pep-0396/

In 3.8+ they added a nicer API for this:
```python
>>> from importlib.metadata import version
>>> print(version('textual'))
0.1.15
```

Reply:
It's going to be a PITA to maintain, and I'd rather not add one of the various hacks to pre-fill this.

How about we use this backport of [importlib-metadata](https://pypi.org/project/importlib-metadata/) ?